--- lib389/cli_conf/replication.py.orig	2020-01-21 01:46:13.000000000 -0500
+++ lib389/cli_conf/replication.py	2020-01-21 01:46:26.000000000 -0500
@@ -644,7 +644,7 @@
         properties['nsds5replicastripattrs'] = args.strip_list
 
     # We do need the bind dn and credentials for none-sasl bind methods
-    if (bind_method == 'simple' or 'sslclientauth') and (args.bind_dn is None or args.bind_passwd is None):
+    if (bind_method in ('simple', 'sslclientauth')) and (args.bind_dn is None or args.bind_passwd is None):
         raise ValueError("You need to set the bind dn (--bind-dn) and the password (--bind-passwd) for bind method ({})".format(bind_method))
 
     # Create the agmt
